herbert_core/utilities/classes/@unique_references_container/unique_references_container.m

```diff
@@ -444,6 +490,54 @@
             nuix = numel(self.idx_);
         end
 
+        function [self, nuix] = add_copies_(self,obj,n)
```

Comment:
This is marked with an `_` but still seems to be a public/non-hidden method.

Reply:
they are within the block of protected methods starting at line 450. It would be more appropriate to put underscores on the rest but for the moment let's avoid breaking code